pkg/agent/datamodel/versions.go

```diff
@@ -373,12 +373,14 @@ func getAllKubernetesWindowsSupportedVersionsMap() map[string]bool {
 }
 
 // GetSupportedVersions get supported version list for a certain orchestrator.
-func GetSupportedVersions(orchType string, isUpdate, hasWindows bool) (versions []string, defaultVersion string) {
+func GetSupportedVersions(orchType string, isUpdate, hasWindows bool) ([]string, string) {
+	versions := []string{}
```

Comment:
can we simplify all this to:

```suggestion
	if osType == Kubernetes {
	    return GetAllSupportedKubernetesVersions(isUpdate, hasWindows), GetDefaultKubernetesVersion(hasWindows)
	}
	return nil, ""
```

or is that something the linter doesn't like